charts/incubator/it-tools/values.yaml

```diff
@@ -0,0 +1,22 @@
+image:
+  repository: tccr.io/truecharts/it-tools
+  pullPolicy: IfNotPresent
+  tag: v2023.8.16@sha256:054bd18916e3cce944bd9e4d7e9d2947b7492ecc401b1dec3cba989d8ca0a1ee
+
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        targetPort: 80
+        port: 18082
+
+persistence:
+  cache:
+    enabled: true
+    mountPath: "/var/cache/nginx"
```

Comment:
https://github.com/truecharts/charts/pull/11303#discussion_r1292388146

Reply:
Got it, same issue. Fixed